### PR TITLE
ejabberd: bump to 21.04 

### PR DESCRIPTION
**Pull request by grueni**
See https://github.com/OpenIndiana/oi-userland/pull/6719

### CODE REVIEW

**Comment by AndWac** [comment]
You'll also need to increment COMPONENT_REVISION in the Makefile otherwise it's not guaranteed that this update will be installed.

File: components/web/ejabberd/Makefile [hunk 1]
```diff
@@ -21,30 +21,31 @@
...
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= ejabberd
 COMPONENT_FMRI= web/server/$(COMPONENT_NAME)
-COMPONENT_VERSION= 17.11
-COMPONENT_REVISION= 1
+# 21.04 will fail with "ValueError: Zero padded number" during mogrify
+COMPONENT_VERSION= 21.4
 COMPONENT_PROJECT_URL=	http://www.ejabberd.im/
```
Comment (AndWac) [inline components/web/ejabberd/Makefile:35]: In this case we have IPS_COMPONENT_VERSION, too. So you can use
COMPONENT_VERSION= 21.04
IPS_COMPONENT_VERSION=21.0.4
(We usually model leading zeroes like this).

Comment (grueni) [reply components/web/ejabberd/Makefile:35]: Thanks, works as it should

File: components/web/ejabberd/files/svc-xmpp-ejabberd [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 #
 # Copyright (c) 2009, 2011, Oracle and/or its affiliates. All rights reserved.
-# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+# Copyright 20212121212121212121212121212121212121212121rueni). All rights reserved.
 #
```
Comment (AndWac) [inline components/web/ejabberd/files/svc-xmpp-ejabberd:26]: Is this strange copyright message intended?

Comment (grueni) [reply components/web/ejabberd/files/svc-xmpp-ejabberd:26]: Not really

**Comment by AndWac** [comment]
Please squash the commits before I merge this PR. While squashing please remove commit messages that are irrelevant.

**Pull request merged**